src/main/java/pl/javastart/task/DataReadWrite.java

```diff
@@ -0,0 +1,97 @@
+package pl.javastart.task;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class DataReadWrite {
+    private static final Scanner sc = new Scanner(System.in);
+
+    public static void motApp() throws IOException {
+        readFile();
+        int option = -1;
+        do {
+            printOptions();
+            option = sc.nextInt();
+            sc.nextLine();
+            if (option == 0) {
+                if (MotQueue.queueSize() == 0) {
+                    return;
+                } else {
+                    saveQueueToTheFile();
+                }
```

Comment:
Jeżeli obsłużymy pojazdy, to też chcemy nadpisać plik, żeby był pusty. W innym przypadku przy kolejnym uruchomieniu programu będziemy obsługiwali te same pojazdy co w poprzedniej sesji programu